### PR TITLE
package: add rm2fb package

### PR DESCRIPTION
**Pull request by raisjn**
this adds rm2fb (ddvk/remarkable2-framebuffer) package, which is used for app compatibility on the rM2.

to run an app, first start `rm2fb-server`, then use `rm2fb-client <app>` to launch the app.

### CODE REVIEW

File: package/rm2fb/rm2fb-server [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+LD_PRELOAD=/opt/lib/librm2fb_server.so.1.0.0 `which remarkable-shutdown`
```
Comment (Eeems) [inline package/rm2fb/rm2fb-server]:
```suggestion
LD_PRELOAD=/opt/lib/librm2fb_server.so.1.0.0 $(which remarkable-shutdown)
```

Comment (Eeems) [reply package/rm2fb/rm2fb-server]: This will make the style check happy.

File: package/rm2fb/package [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+maintainer="raisjn <of.raisjn@gmail.com>"
+license=MIT
+pkgdesc="Remarkable2 Framebuffer"
+url="https://github.com/ddvk/remarkable2-framebuffer"
+pkgver=1.0.0-1
+section=utils
+version=1.0.0
+
+
+image=qt:v1.1
```
Comment (Eeems) [inline package/rm2fb/package]:
```suggestion
version=1.0.0

image=qt:v1.1
```
Looks like it also doesn't like the extra newline.

**Comment by Eeems** [comment]
Since I don't have a rM2 I can't test. Anybody else able to test the builds?

**Comment by lcpriest** [comment]
Running `make repo` on this pr threw an error for me.
```
{16:01}[2.5.1]~/Development/toltec:rmkit-dev/testing ✓ ➭ make repo
./scripts/repo-build package build/package build/repo "$remote_repo"
==> Processing appmarkable
==> Processing calculator
==> Processing chessmarkable
==> Processing draft
==> Processing fingerterm
==> Processing koreader
==> Processing oxide
==> Processing plato
==> Processing reboot-guard
==> Processing recrossable
==> Processing remarkable-splash
==> Processing retris
==> Processing rm2fb
date: illegal option -- -
usage: date [-jnRu] [-d dst] [-r seconds] [-t west] [-v[+|-]val[ymwdHMS]] ...
            [-f fmt date | [[[mm]dd]HH]MM[[cc]yy][.ss]] [+format]
make: *** [repo] Error 1
```

If someone can give me some tips on what I should run to properly test this, I can run it on my rm2 if I can get past this blocker.

**Comment by raisjn** [comment]
for testing, see https://github.com/ddvk/remarkable2-framebuffer/issues/35

the build system for PRs also attaches the packages built as artifacts: it's at the bottom of the build check page: https://github.com/toltec-dev/toltec/suites/1697155072/artifacts/31910889 (for the most recent run)

for building, can you share what version of `date` you have? (use `date --version`)

**Comment by raisjn** [comment]
i think enough people (5+) have installed this package and had it work for them

**Review by Eeems — APPROVED** [review]
Due to community testing I'll approve.

**Pull request merged**